packages/scandipwa/src/route/ForgotPassword/ForgotPassword.container.js

```diff
@@ -36,15 +36,21 @@ export const mapDispatchToProps = (dispatch) => ({
 export class ForgotPasswordContainer extends MyAccountOverlayContainer {
     static propTypes = {
         ...MyAccountOverlayContainer.propTypes,
-        updateBreadcrumbs: PropTypes.func.isRequired
+        updateBreadcrumbs: PropTypes.func.isRequired,
+        isForgotPasswordPage: PropTypes.bool
+    };
+
+    static defaultProps = {
+        isForgotPasswordPage: true
     };
 
     containerProps() {
-        const { device } = this.props;
+        const { device, isForgotPasswordPage } = this.props;
 
         return {
             ...super.containerProps(),
-            device
+            device,
+            isForgotPasswordPage
```

Comment:
Is it even necessary to set prop if you can just set argument in function above to false and pass it to component where you need it?